app/models/katello/content_view_version.rb

```diff
@@ -109,6 +109,14 @@ def description
       history.publish.successful.first.try(:notes)
     end
 
+    def description=(new_description)
+      if description
```

Comment:
You might want to change this to `if history.publish.successful.first`

If the description were null for some reason we'd still want to set it.